github-pr

```diff
@@ -12,11 +12,19 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    commit_statuses = []
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            commit_statuses.append(commit_status.state)
+    return commit_statuses[-1]
```

Comment:
I still don't think this is right. If I have a Pr with 2 commits, the first one passed all testing, the second one was never tested or has not finished testing, this will return that the Pr is passing.